entries/w/wp.pl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
+    "tfa": [
+      "totp",
+      "custom-software"
```

Comment:
Could you please specify the custom software?

Reply:
It's https://play.google.com/store/apps/details?id=pl.wp.onelogin
It supports TOTP offline mode and online mode where the user confirms login requests in the app.

Reply:
Please could you specify this with a `custom-software` field, like below:
```JSON
"custom-software": [
  "1login od WP"
]
```
There is more information on this in the [Other Properties](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties) section of the contributing guidelines.